### PR TITLE
Add documantation to Share component

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?
As title says

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Share/README.md [hunk 1]
```diff
@@ -1,40 +1,74 @@
 # Share
 
-Share is a Component that will allow you to configure share (social media and email) buttons.
+## Description
 
-And to import it into your code:
+`Share` is a VTEX component that allows to share a product url via social medias.
+This Component can be imported and used by any VTEX app.
```
Comment (paul0vinicius) [inline react/components/Share/README.md]:
```suggestion
This component can be imported and used by any VTEX app.
```

File: react/components/Share/README.md [hunk 2]
```diff
@@ -1,40 +1,74 @@
...
+#### Configuration
+
+Through the Storefront, you can change the `Share`'s behavior and interface. However, you also can make in your theme app, as Store theme does. However, you also can make in your theme app, as Store theme does.
+
+| Prop name | Type | Description | Default value |
+| --------- | ---- | ----------- | ------------- |
+| `className` | `String` | The main container classes | null |
+| `shareLabelClass` | `String` | The share label classes | true |
+| `buttonsContainerClass` | `String` | The button container classes | true |
+| `options` | `Object` | Share button options, like "size" | {} |
```
Comment (GustavoMA) [inline react/components/Share/README.md]:
```suggestion
| `options` | `Options` | Share button options, like "size" | {} |
```

File: react/components/Share/README.md [hunk 3]
```diff
@@ -1,40 +1,74 @@
...
-| `shareLabelClass` | `String` | Classes to be applied to Share label |
-| `socialButtonClass` | `String` | Classes to be applied to social button |
-| `socialIconClass` | `String` | Classes to be applied to icon of the button |
-| `loaderContainerClass` | `String` | Classes to be applied to the `ContentLoader` container |
-| `contentLoaderClass` | `String` | Classes to be applied to the `ContentLoader` |
+For now this block does not have any required or optional blocks.
+
+#### Configuration
+
+Through the Storefront, you can change the `Share`'s behavior and interface. However, you also can make in your theme app, as Store theme does. However, you also can make in your theme app, as Store theme does.
```
Comment (paul0vinicius) [inline react/components/Share/README.md]:
```suggestion
Through the Storefront, you can change the `Share`'s behavior and interface. However, you also can make in your theme app, as Store theme does. 
```

File: react/components/Share/README.md [hunk 4]
```diff
@@ -1,40 +1,74 @@
...
+
+| Prop name | Type | Description | Default value |
+| --------- | ---- | ----------- | ------------- |
+| `className` | `String` | The main container classes | null |
+| `shareLabelClass` | `String` | The share label classes | true |
+| `buttonsContainerClass` | `String` | The button container classes | true |
+| `options` | `Object` | Share button options, like "size" | {} |
+| `social` | `Object` | The possible social medias to be displayed | {Facebook: true, Twitter: true, WhatsApp: true} |
```
Comment (GustavoMA) [inline react/components/Share/README.md]:
```suggestion
| `social` | `Social` | The possible social medias to be displayed | {Facebook: true, Twitter: true, WhatsApp: true} |
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Add an API Docs to `ProductSpecifications`
 - Add an API Docs to `GradientCollapse`.
 - Add an API Docs to `CategoriesHighlights`.
+- Add an API DOcs to `Share`
```
Comment (JuanAlmeida12) [inline CHANGELOG.md]: Add dot at end.

**Pull request merged**